code/data_preprocessing/genotype/genotype_formatting.ipynb

```diff
@@ -388,7 +388,7 @@
     "[plink_by_gene_2]\n",
```

Comment:
<p>this is not intended right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cumc/xqtl-pipeline/pull/451/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
This is intended due to #454, I have tried to reproduce the error consistently but fail, so I decided to patch it this way as they don't need to be run with the task statement in the first place.